docker-compose/db-only-migrate.docker-compose.yaml

```diff
@@ -30,7 +30,7 @@ services:
 
   codeintel-db:
     container_name: codeintel-db
-    image: 'index.docker.io/sourcegraph/codeintel-db:5.10.0@sha256:5df73dbaf7714e7a9ccaeb1d2f9ec6a84e9f746ec2ef46296d95bb693bad8ffe'
+    image: 'index.docker.io/sourcegraph/codeintel-db:5.10.2832@sha256:019bb4090f2bc20218887f5fd1bcfaa122ca090460f5803e1df4f6637947cb26'
```

Comment:
@jdpleiness should we update these to be postgres-16 image? I don't recall what these db-only compose files are for.

Reply:
Yeah these should be updates as well. From the comments:

```
# This file contains the pgsql service definition (the primary Sourcegraph database deployment), 
# as well as the codeintel-db service definition (a separate postgres deployment). Both run postgres.
# This file is used during the sourcegraph/server -> docker-compose migration process.
```